motr/client.c

```diff
@@ -1011,20 +1040,21 @@ int m0_client_calculate_pi(struct m0_generic_pi *pi,
 		struct m0_pi_seed *seed,
 		struct m0_bufvec *bvec,
 		enum m0_pi_calc_flag flag,
-		unsigned char *curr_digest,
+		unsigned char *curr_context,
 		unsigned char *pi_value_without_seed)
 {
 	M0_ENTRY();
 
 	switch(pi->hdr.pi_type)
 	{
```

Comment:
Place the opening brace on previous line.

Reply:
done.